docs/app-debug/valgrind.rst

```diff
@@ -0,0 +1,24 @@
+Using Valgrind to Find Open MPI Application Errors
+==================================================
+
+The `Valgrind <https://valgrind.org/>` utilities may be useful in identify application
+logic errors Howver, there are many situations where Open MPI purposefully does
```

Comment:
Fix a couple of typo, and one formatting issue in the URL.
```suggestion
The `Valgrind <https://valgrind.org>`_ utilities may be useful in identifying application
logic errors. However, there are many situations where Open MPI purposefully does
```

Reply:
I don't see these changes in the latest push.

The `_` after the URL is needed for it to render as a link. Currently it renders as italic text
![image](https://user-images.githubusercontent.com/4259120/204328915-672501b6-6e06-4ace-b0e8-1e97f23f6ab2.png)